### ISSUE TITLE
Adding stencil-ionic version

### ISSUE THREAD

**Issue by jgw96** [tastejs/hacker-news-pwas#111]
Hey all! Just opened a PR to add the [Stencil](https://stenciljs.com/) version https://github.com/tastejs/hacker-news-pwas/pull/110. Hoping i did everything correct. Only thing i was not sure about was the `weight` property.

**Comment by addyosmani** [comment]
Thanks for the PR, @jgw96! We're excited to take a look at it.

**Comment by jgw96** [comment]
Thanks! Let me know if there is anything I need to tweak (:

**Comment by kristoferbaxter** [comment]
Leaving this out of the PR since it shouldn't impact submission.

First, awesome addition... I'm excited about stencil and glad to see a HN PWA using it.

Secondly, I noticed something strange in the Webpage Test results yesterday, so I ran another test today to confirm. 

https://www.webpagetest.org/video/compare.php?tests=170914_R0_bedac1f8757fb28bd9f18daae5fa9e46-r:1-c:0

![screen shot 2017-09-14 at 11 06 25 am](https://user-images.githubusercontent.com/61764/30446021-cc2b2f1a-993c-11e7-84f5-fcfed39f7c77.png)

Visual progress stays below 75% until ~6.8s. From ~2.5s to ~6.8s the user is shown a loading state.

![screen shot 2017-09-14 at 11 07 36 am](https://user-images.githubusercontent.com/61764/30446081-f3e4efc8-993c-11e7-832d-fe1198d516a0.png)

Improving this would lead to a better experience for users on first load scenarios, so it might be worth thinking about.
